src/components/ItemList/ItemList.module.css

```diff
@@ -2,3 +2,15 @@
   list-style-type: none;
   text-align: left;
 }
+
+.buttons {
+  display: flex;
+  justify-content: space-around;
+}
+
+@media (max-width: 1080px) {
+  .list {
+    width: 70%;
+    margin: 0 auto;
+  }
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/ReactProject_karolina/pullRequest?prid=6313319)